src/test/java/com/codingnomads/betty/data/batch/batchwriters/MatchOddsRdsWriterTests.java

```diff
@@ -24,15 +25,16 @@ public void setUp(){
         testMatchOddsRdsWriter = new MatchOddsRdsWriter(mockMatchOddsJpaRepository);
         matchOdds = new MatchOdds();
         list = Arrays.asList(matchOdds);
+        items = Arrays.asList(list);
```

Comment:
Same here as above... You should use a singleton list here to prevent accidental nullpointers. Consider changing 'Arrays.asList' to 'Collections.singletonList'